src/vtbackend/Settings.h

```diff
@@ -89,6 +89,7 @@ struct Settings
     PrimaryScreen primaryScreen;
 
     bool fromSearchIntoInsertMode = true;
+    bool inInsertAfterYank = true;
```

Comment:
Here it defaults to `true`, but above (in `src/contour/Config.h`), the value is defaulted to `false`. I think it should both be the same (my preference would be `false`, to retain current semantics and standard vi-experience).

Reply:
You are right, I was using it for testing purposes, I still need to connect config and settings